build.gradle

```diff
@@ -7,23 +7,26 @@ plugins {
 group = 'com.thoxia'
 version = '1.0.0'
 
-repositories {
-    mavenCentral()
-    maven {
-        name = "papermc-repo"
-        url = "https://repo.papermc.io/repository/maven-public/"
-    }
-    maven {
-        name = "sonatype"
-        url = "https://oss.sonatype.org/content/groups/public/"
+
+    repositories {
+        mavenCentral()
+        maven {
+            name = "papermc-repo"
+            url = "https://repo.papermc.io/repository/maven-public/"
+        }
+        maven {
+            name = "sonatype"
+            url = "https://oss.sonatype.org/content/groups/public/"
+        }
+        maven { url "https://storehouse.okaeri.eu/repository/maven-public/" }
+        maven { url 'https://repo.triumphteam.dev/snapshots/' }
+        maven { url 'https://jitpack.io' }
+        maven { url 'https://repo.xenondevs.xyz/releases' }
+        maven { url 'https://repo.extendedclip.com/content/repositories/placeholderapi/' }
+        maven { url 'https://repo.essentialsx.net/releases/' }
+        mavenCentral()
     }
```

Comment:
Could you please revert these changes? It's not in the right format for this file.
You just need to format the spaces correctly, other than that, looks good to me.